psbt/src/lib.rs

```diff
@@ -89,3 +89,27 @@ impl PsbtVer {
         }
     }
 }
+
+impl std::convert::TryFrom<usize> for PsbtVer {
```

Comment:
`PsbtVer` already implements `TryFrom<u8>` (via `#[strict_type(try_from_u8)` on the type), so we do not need this convertor

Reply:
No, it did not have this trait.
https://github.com/BP-WG/bp-std/blob/f4b1861fd809ae28d364a9e79162ec4294f9de74/psbt/src/lib.rs#L60-L66

I tried and make sure the trait you mentioned did not exist.
 
```console
error[E0277]: the trait bound `PsbtVer: From<u8>` is not satisfied
   --> psbt/src/data.rs:550:30
    |
550 |                 Some(ver) => PsbtVer::try_from(ver as u8).map_err(|_| fmt::Error)?,
    |                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `From<u8>` is not implemented for `PsbtVer`, which is required by `PsbtVer: TryFrom<u8>`
    |
    = note: required for `u8` to implement `Into<PsbtVer>`
    = note: required for `PsbtVer` to implement `TryFrom<u8>`
```